src/pages/ContactsItemEdit/components/ContactEditForm/ContactEditForm.js

```diff
@@ -0,0 +1,136 @@
+// @flow strict
+
+import React, { PureComponent } from 'react'
+import { withI18n } from '@lingui/react'
+import { type I18n as I18nType } from '@lingui/core'
+
+import {
+  Form,
+  Field,
+  type FormRenderProps,
+} from 'react-final-form'
+
+import {
+  Button,
+  JInputField,
+} from 'components/base'
+
+import offset from 'styles/offsets.m.scss'
+import style from './contactEditForm.m.scss'
+
+export type FormValues = {|
+  address: string,
+  description: string, // note
+  name: string,
+|}
+
+export type Props = {|
+  +initialValues: FormValues,
+  +i18n: I18nType,
+  +checkContactExists: (address: OwnerAddress) => boolean,
+  +editContact: (contact: FormValues) => any,
+  +onDeleteClick: (address: OwnerAddress) => any,
```

Comment:
I suggest to name handlers in the same way, e.g. editContact/deleteContact or onEdit/onDelete, etc

Reply:
I don't agree. There is strong difference between edit and click on edit button. `onEdit` could mean: on click on edit button, on type in input, on change input, on focus in input and so on.

Reply:
I also suggest we put handler first, description second. So it would all be structured the same way for all the handlers, like: `onClick` (if it is the only element), `onClickEdit`, `onClickDelete`, `onChangeName`, `onFocusIcon` etc.